presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -350,8 +351,12 @@ protected RelationType visitDescribeInput(DescribeInput node, AnalysisContext co
         // get all parameters in query
         List<Parameter> parameters = getParameters(statement);
 
-        // return the positions and types of all parameters
+        // return the positions and types of all parameters.  If there are no parameters, return a single null row.
         Row[] rows = parameters.stream().map(parameter -> createDescribeInputRow(parameter, queryAnalysis)).toArray(Row[]::new);
+        if (rows.length == 0) {
+            rows = new Row[] {row(new NullLiteral(), new NullLiteral())};
```

Comment:
You were saying in person that it would be difficult to create a row with empty strings. Why can't you create a row here with two `new StringLiteral("")`? Or maybe I misunderstood you ...